### DISCUSSION TITLE
Issues with Triggering Spotify Player Using Hyperaudio Plugin

### DISCUSSION THREAD

**Discussion by ataha24** [hyperaudio/wordpress-hyperaudio#144] [General]
I appreciate the recent update that allows linking transcripts with Spotify in the hyperaudio plugin. However, I'm encountering difficulties in activating the Spotify player from my embedded code. The documentation does not provide specific instructions for this integration, so I assumed the standard syntax would apply.

Here's what I've attempted:

1. I retrieved the embed code for my media from Spotify.
2. I linked the transcript on my website using two methods:

a) With the player specified:
`[hyperaudio src="https://open.spotify.com/embed/episode/0J7C9SIIwQvARa9WDmtOxB" font-family="Arial" show-active="true" player="Spotify"]`

b) Wihout player specified: 
`[hyperaudio src="https://open.spotify.com/embed/episode/0J7C9SIIwQvARa9WDmtOxB" font-family="Arial" show-active="true"]`

Unfortunately, neither approach triggered the Spotify player. Attached is a screenshot showing the output. It seems that the hyperaudio plugin might not be properly invoking the Spotify player.

Could you please advise on the correct approach or any additional steps I need to take? If there's any further information you need, I'm happy to provide it.

Thank you for your assistance.

<img width="543" alt="Screenshot 2023-12-07 at 9 27 25 AM" src="https://github.com/hyperaudio/wordpress-hyperaudio/assets/46094728/f8f0e9a5-4463-4419-ad67-c3824abcf28a">

**Comment by maboa** [comment]
Hi @ataha24. Try removing `/embed` from your Spotify URLs. Hope it's ok if I move this to discussions so others with similar issues can benefit from this answer.

**Reply by ataha24** [reply]
Thanks for the constant support and apologies for the delayed response. Unfortunately, removing the /embed from the URL did not do it. Is there another recommended fix? This is still of great interest!

**Reply by ataha24** [reply]
Hi, I wanted to follow-up re this issue. Thanks!

**Reply by maboa** [reply]
You need to make sure it is categorised as an episode ie `https://open.spotify.com/episode/4XAhcdJTXoffuDZglrEbSj` Feel free to share a link to the url you are trying to embed and I will try and debug.

**Reply by ataha24** [reply]
I see! I think the one I am working with is tagged as an episode. Here is the link to it: https://open.spotify.com/episode/0J7C9SIIwQvARa9WDmtOxB. Thank you so much!

**Reply by maboa** [reply]
This seems to be working for me @ataha24 . 
![Screenshot 2024-02-26 at 16 55 21](https://github.com/hyperaudio/wordpress-hyperaudio/assets/208756/b9387e2b-6e6e-480c-af94-265d296c4e1c)
Try modifying this Spotify example to include your episode and see how you get on. https://hyperaudio.github.io/hyperaudio-lite/spotify.html (https://github.com/hyperaudio/hyperaudio-lite/blob/main/spotify.html)

**Reply by ataha24** [reply]
Ahh, yes will play around there as recommended and see what happens! 
Though, I am interfacing with the wp-plugin so wondering if there was an issue with how I configued it there (please see above re unexpected behaviour with player on the wp side of things).
Thanks :)
